src/Shape.jl

```diff
@@ -236,8 +236,12 @@ function prepare_to_interpolate(from_shape, to_shape)
     end
 
     # rotate outer polygon
-    rotate_i, _ = compute_shortest_morphing_dist(from_outer, to_outer)
-    new_from_outer = circshift(from_outer, -rotate_i + 1)
+    if style == :short
+        rotate_i, _ = compute_shortest_morphing_dist(from_outer, to_outer)
+        new_from_outer = circshift(from_outer, -rotate_i + 1)
+    elseif style == :long
```

Comment:
doesn't this depend on the ordering of the initial polygons? Would it make sense to circshift according to shortest first and then reverse that?